Add 'We are not accepting apps' to the docs and tutorials (#6318)
PostHog/posthog.com: contents/docs/cdp/build/index.md

showTitle: true
---

> > **Note:** You can create apps for both self-hosted and PostHog Cloud, but releasing an app on PostHog Cloud requires a review process. [Read more about the review process in the tutorial](/docs/apps/build/tutorial#submitting-your-app). We are temporarily not reviewing new apps while we build a new export system.

Reviewer: ```suggestion
> **Note:** You can create apps for self-hosted PostHog instances.
```

> > **Note:** You can create apps for both self-hosted and PostHog Cloud, but releasing an app on PostHog Cloud requires a review process. [Read more about the review process in the tutorial](/docs/apps/build/tutorial#submitting-your-app). > We are currently not reviewing new apps while we build a new export system. We will update these docs with more information as that work is completed. 

Reviewer 2: ```suggestion
> **Note:** You can create apps for both self-hosted and PostHog Cloud, but releasing an app on PostHog Cloud requires a review process. [Read more about the review process in the tutorial](/docs/apps/build/tutorial#submitting-your-app). 

> We are currently **not reviewing new apps** while we build [a new export system](https://github.com/PostHog/posthog/issues/15997). We will update these docs with more information as that work is completed. 
```
